src/main/java/gov/cabinetoffice/gap/applybackend/service/SubmissionService.java

```diff
@@ -215,6 +222,17 @@ public void submit(final Submission submission, final GrantApplicant grantApplic
         }
 
         submission.setGapId(generateGapId(grantApplicant.getId()));
+        try {
+            final UUID submissionId = submission.getId();
+            final Optional<GrantMandatoryQuestions> grantMandatoryQuestion = ofNullable(grantMandatoryQuestionRepository.findBySubmissionId(submissionId)
```

Comment:
I have a feeling this won't work with V1 schemes as it will always throw a not found exception when trying to submit because those schemes don't require MQs to be completed. Can you test this and get back to me pls? Or we can test it together if you're not set up with admin accounts etc